static/sass/styles.scss

```diff
@@ -772,3 +774,10 @@ body {
   right: 0;
   top: 0;
 }
+
+.article-title {
+  -webkit-box-orient: vertical;
+  display: -webkit-box;
```

Comment:
I've never seen this display property before nor can I find anything on it by Googling.. what does it do? Does autoprefixer polyfill not webkit browsers?

Reply:
It a webkit only property which allows us to use the new line-clamp (-webkit-line-clamp). Which is the only CSS only way to truncate lines over more than one line. https://css-tricks.com/line-clampin/